client/extensions/woocommerce/app/dashboard/manage-orders.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import { getLink } from '../../lib/nav-utils';
```

Comment:
Agreed! Fixed in https://github.com/Automattic/wp-calypso/pull/14339/commits/fc929a9089c59738059bec4d882378d93712696e